blog/2023-01-10-pulsar-2022-year-in-review.md

```diff
@@ -0,0 +1,109 @@
+
+- [Pulsar Summit San Francisco 2022](https://streamnative.io/blog/community/2022-08-25-pulsar-summit-sf-2022-community-event-recap/): This event marked the first-ever, in-person Pulsar Summit in North America with 200+ attendees. It featured 20 speakers from companies like Google, AWS, and Yahoo!, 5 keynotes on Apache Pulsar and other streaming technologies, and 12 breakout sessions on tech deep dives, use cases, and ecosystem talks. [Watch the sessions](https://streamnative.io/pulsar-summit-on-demand/).
+- [Pulsar Summit Asia 2022](https://pulsar.apache.org/blog/2022/12/01/pulsar-summit-asia-2022-recap/): With 1,500+ registrations and 40,000+ views globally, this two-day virtual event gathered 41 speakers giving 3 keynote addresses and 36 sessions. They came from companies like ByteDance, Huawei, Tencent, Nippon Telegraph and Telephone Corporation (NTT), Yum China, Netease, vivo, Nutanix, and StreamNative. [Watch the sessions](https://www.youtube.com/playlist?list=PLqRma1oIkcWgeNb3kgzqFyg5sywH8S4yy).
+
+To stay up to date with the latest events, you can join the following meetups or groups:
+
+- [New York City Apache Pulsar Meetup](https://www.meetup.com/new-york-city-apache-pulsar-meetup/)
+- [SF Bay Area Apache Pulsar Meetup](https://www.meetup.com/sf-bay-area-apache-pulsar-meetup/)
+- [Netherlands Apache Pulsar Meetup](https://www.meetup.com/netherlands-apache-pulsar-meetup/)
+- [Japan Pulsar User Group](https://japan-pulsar-user-group.connpass.com/)
+
+## Community growth
+
+The development of the Pulsar community would not be possible without our contributors. Among them, Pulsar Committers and PMC members have taken the lead in contributing to and promoting the project. In 2022, many new faces joined the community while we also welcomed old friends taking more responsibility. Let’s take a look at the Pulsar community by numbers.
+
+- **598** global contributors for the Pulsar main GitHub repository
+- **12.1K+** GitHub stars
+- **3.2K+** forks
+- **16** new Committers
+- **5** new PMC members
+- **9560** Pulsar Slack members
+- **4.3K+** downloads
+- **14M+** Docker pulls
+
+The Pulsar community welcomes all kinds of contributions. For more information, see the [Apache Pulsar Contribution Guide](https://pulsar.apache.org/contribute/).
+
+## Project releases
+
+In 2022, the Pulsar community worked hard to improve the project’s capabilities and fix existing bugs with 1 major version and 8 minor versions. Many important Pulsar capabilities were delivered in these releases, such as [cluster-level automatic failure recovery](https://github.com/apache/pulsar/pull/13316) and [lazy-loading added to PartitionedProducer](https://github.com/apache/pulsar/pull/10279). Check out each release below to see details.
+
+- [2.7.5](https://github.com/apache/pulsar/releases/tag/v2.7.5)
```

Comment:
Use reverse order here? Then users can read the latest versions first.

Reply:
Changed.